deploy/charts/cert-manager/templates/rbac.yaml

```diff
@@ -267,27 +267,6 @@ rules:
 
 ---
 
-apiVersion: rbac.authorization.k8s.io/v1beta1
-kind: ClusterRoleBinding
-metadata:
-  name: {{ template "cert-manager.fullname" . }}-leaderelection
-  labels:
-    app: {{ template "cert-manager.name" . }}
-    app.kubernetes.io/name: {{ template "cert-manager.name" . }}
-    app.kubernetes.io/instance:  {{ .Release.Name }}
-    app.kubernetes.io/managed-by: {{ .Release.Service }}
-    helm.sh/chart: {{ template "cert-manager.chart" . }}
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: {{ template "cert-manager.fullname" . }}-leaderelection
```

Comment:
Where is this created? I'm trying to deploy via Helmfile and it's complaining that this doesn't exist. I'm sure if Helmfile is doing something different, but I don't encounter the error when deploying directly via Helm

```
Error: release cert-manager failed: clusterroles.rbac.authorization.k8s.io "cert-manager-leaderelection" not found
```